numpy/compat/py3k.py

```diff
@@ -10,6 +10,10 @@
            'integer_types']
 
```

Comment:
Add `Path` and `is_pathlib_path` to the `__all__` list, then you can import from `numpy.compat`.